CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 CHANGELOG
 =========
 
+# 0.31.0 / 2019-10-30
+
+* [FEATURE] Add support for SLOs [#453][] and [#464][]
+* [FEATURE] Add ability to send compressed payloads for metrics and distribution. [#466][]
+* [FEATURE] Add parameter to enable/disable hostname lookup from datadog-agent config to avoid warnings [#428][]
+* [FEATURE] Dogstatsd: add ability to specify a default sample rate for all submissions [#470][] (thanks [@dtao][])
+* [BUGFIX] Do not leak file descriptor when reading config [#425][]
```

Comment:
```suggestion
* [BUGFIX] Fix possible issue that could leak file descriptors when reading config [#425][]
```

Reply:
Many thanks. I just could not figure out a good way to phrase it